rust/ql/test/library-tests/dataflow/strings/main.rs

```diff
@@ -56,7 +56,7 @@ fn string_from() {
 fn string_to_string() {
     let s1 = source_slice(22);
     let s2 = s1.to_string();
-    sink(s2); // $ MISSING: hasTaintFlow=22 - we are not currently able to resolve the `to_string` call above, which comes from `impl<T: fmt::Display + ?Sized> ToString for T`
+    sink(s2); // $ MISSING: hasTaintFlow=22
```

Comment:
The explanation here was outdated. We actually resolve the call now, but we can't model the method in a blanket implementation.